sky/cli.py

```diff
@@ -1370,24 +1371,40 @@ def status(all: bool, refresh: bool):  # pylint: disable=redefined-builtin
 
     Display all fields using ``sky status -a``.
 
-    \b
     Each cluster can have one of the following statuses:
 
-    \b
-    - INIT: The cluster may be live or down. It can happen in following cases:
-      (1) undergoing provisioning or runtime setup. (In other words, a
-      ``sky launch`` has started but has not completed.)
-      (2) Or, the cluster is in an abnormal state, e.g., some cluster nodes are
-      down, or the sky runtime has crashed.
-    - UP: Provisioning and runtime setup have succeeded and the cluster is
+    * ``INIT``: The cluster may be live or down. It can happen in the following
+      cases:
+
+      * Ongoing provisioning or runtime setup. (A ``sky launch`` has started
+        but has not completed.)
+
+      * Or, the cluster is in an abnormal state, e.g., some cluster nodes are
+        down, or the SkyPilot runtime is unhealthy.
+
+    * ``UP``: Provisioning and runtime setup have succeeded and the cluster is
       live.  (The most recent ``sky launch`` has completed successfully.)
-    - STOPPED: The cluster is stopped and the storage is persisted. Use
+
+    * ``STOPPED``: The cluster is stopped and the storage is persisted. Use
       ``sky start`` to restart the cluster.
 
-    The autostop column indicates how long the cluster will be autostopped
-    after minutes of idling (no jobs running). If the time is followed by
-    '(down)', e.g. '1m (down)', the cluster will be autodowned, rather than
-    autostopped.
+    Autostop column:
+
+    * The autostop column indicates how long the cluster will be autostopped
+      after minutes of idling (no jobs running). If the time is followed by
+      '(down)', e.g. '1m (down)', the cluster will be autodowned, rather than
+      autostopped.
+
+    Getting up-to-date cluster statuses:
+
+    * In normal cases where clusters are entirely managed by SkyPilot (i.e., no
```

Comment:
This is a very clear decision boundrary 👍 Can we say in general Skypilot tracks the state of the cluster whenever a Sky is called but between periods of Sky invocation, any modification to the state will not be tracked?

Reply:
This  paraphrasing is good, and that's what the current PR says, plus some common scenario examples.